backend/prompt_studio/prompt_studio_output_manager/views.py

```diff
@@ -59,21 +61,16 @@ def get_output_for_tool_default(self, request: HttpRequest) -> Response:
         # Get the tool_id from request parameters
         tool_id = request.GET.get("tool_id")
         document_manager_id = request.GET.get("document_manager")
-
+        tool_validation_message = PromptOutputManagerErrorMessage.TOOL_VALIDATION
         if not tool_id:
-            return Response(
-                {"error": "tool_id parameter is required"},
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+            logger.error(f"Validation error: {tool_validation_message}")
```

Comment:
```suggestion
```
We can avoid logging and raising exceptions since the middleware helps with it, also such validation errors are automatically raised while using a serializer.